src/errors/404.njk

```diff
@@ -10,6 +10,7 @@ ignore_in_sitemap: true
 
     <p class="govuk-body">If you typed the web address, check it is correct.</p>
     <p class="govuk-body">If you pasted the web address, check you copied the entire address.</p>
+    <p class="govuk-body">If you are looking for how to implement the 'page not found' pattern see <a class="govuk-link" href="/patterns/page-not-found-pages/">page not found pages</a>.</p>
```

Comment:
I think you can simplify to 'If you're looking for the 'page not found' pattern ...' (with bonus style guide contraction)